src/flows/pipes/Notification/endpoints/Mailgun/index.ts

```diff
@@ -33,37 +37,57 @@ export default register => {
 	|> monitor["notify"](
     data: notification,
     endpoint: ((r) => {
-        apiKey = secrets.get(key: "${data.apiKey}")
-        http.post(
-            url: "https://api.mailgun.net/v3/${data.domain}/messages",
-            headers: {
-              "Content-type": "application/json",
-              "Authorization": "Basic api:\${apiKey}"
-            },
-            data: bytes(v: "{
-              \\"from\\": \\"Username mailgun@${data.domain}\\",
-              \\"to\\": \\"${data.email}\\",
-              \\"subject\\": \\"InfluxDB Alert\\",
-              \\"text\\": \\"\${ r._message }\\"
-            }"))
-    })
-  )`,
-    generateTestQuery: data => `
-    apiKey = secrets.get(key: "{data.apiKey}")
-    http.post(
-        url: "https://api.mailgun.net/v3/${data.domain}/messages",
+      apiKey = secrets.get(key: "${data.apiKey}")
+      auth = http.basicAuth(u: "api", p: "\${apiKey}")
+      url = "https://api.mailgun.net/v3/${data.domain}/messages"
+      data = strings.joinStr(arr: [
+          "from=${fromEmail}",
+          "to=${data.email}",
+          "subject=${subject}",
+          "text=\${r._message}"
+        ], v: "&"
+      )
+      http.post(
+        url: url,
         headers: {
-          "Content-type": "application/json",
-          "Authorization": "Basic api:\${apiKey}"
+          "Content-Type": "application/x-www-form-urlencoded",
+          "Authorization": "\${auth}"
```

Comment:
```suggestion
          "Authorization": auth
```
Same question as above. I'm not sure about the syntax that if `\` will make the variable to be hard coded string here.

Reply:
I am okay with keeping it like this or changing both. As it was like this from the beginning, that suggests that we actually want it to be as is.